main.tf

```diff
@@ -1,3 +1,23 @@
-resource "random_pet" "template" {
-  length = var.length
+locals {
+  dd_team = {
+    for member in var.users :
+    member.username => member if member.access_roles["datadog"]
+  }
+}
+
+data "datadog_role" "standard" {
+  filter = "Datadog Standard Role"
+}
+
+resource "datadog_user" "users" {
+  for_each             = local.dd_team
+  disabled             = each.value.disabled
+  email                = each.value.email
+  name                 = each.value.name
+  roles                = [data.datadog_role.standard.id]
```

Comment:
We should work to provide some mapping to the other roles as well -- Just including the standard feels like we're not providing a complete module since we know we've used the others elsewhere. Mind adding that?

Reply:
@Gowiem the reason I didn't add the mapping is that currently we use the following schema for access_roles:
```
  - username: veronika.gnilitska
    name: Veronika Gnilitska
    ...
    access_roles:
      aws: false
      datadog: true
```
So datadog is just enabled or disabled.
I can update this module to support out-of-the-box roles, something like:
```
    access_roles:
      datadog: 
         enabled: true
         role: [standard | ro | admin]
```
And after that - updated our team.yaml.
How does that sound?

Reply:
Added support for the out-of-the-box roles. Let me know your thoughts.

Reply:
Ah, this `access_roles` shouldn't be here at all. I'll push some changes soon.

Reply:
@Gowiem okay, I think I cleaned this out.
I'd love to work on tests and things like that, let's discuss the priority on our next call.

Reply:
Ah sounds good!